code/src/Form.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+
+import { Header } from './Header'
+import { FirstQuestion } from './FirstQuestion'
```

Comment:
Could it be even clearer if you had named each question-component alike? Like question1, question2 on so on?